pmd-java/src/main/java/net/sourceforge/pmd/lang/java/rule/AbstractLombokAwareRule.java

```diff
@@ -15,29 +17,38 @@
 import net.sourceforge.pmd.lang.java.ast.ASTEnumDeclaration;
 import net.sourceforge.pmd.lang.java.ast.ASTImportDeclaration;
 import net.sourceforge.pmd.lang.java.ast.ASTName;
+import net.sourceforge.pmd.lang.java.ast.Annotateable;
+
 
 /**
  * Base class for rules, that should ignore classes/fields that are annotated
  * with Lombok annotations.
  *
  * @author Andreas Dangel
  */
-public class AbstractLombokAwareRule extends AbstractJavaRule {
+public class AbstractLombokAwareRule extends AbstractIgnoredAnnotRule {
 
     private boolean lombokImported = false;
     private boolean classHasLombokAnnotation = false;
     private static final String LOMBOK_PACKAGE = "lombok";
     private static final Set<String> LOMBOK_ANNOTATIONS = new HashSet<>();
 
     static {
-        LOMBOK_ANNOTATIONS.add("Data");
-        LOMBOK_ANNOTATIONS.add("Getter");
-        LOMBOK_ANNOTATIONS.add("Setter");
-        LOMBOK_ANNOTATIONS.add("Value");
-        LOMBOK_ANNOTATIONS.add("RequiredArgsConstructor");
-        LOMBOK_ANNOTATIONS.add("AllArgsConstructor");
-        LOMBOK_ANNOTATIONS.add("NoArgsConstructor");
-        LOMBOK_ANNOTATIONS.add("Builder");
+        LOMBOK_ANNOTATIONS.add("lombok.Data");
+        LOMBOK_ANNOTATIONS.add("lombok.Getter");
+        LOMBOK_ANNOTATIONS.add("lombok.Setter");
+        LOMBOK_ANNOTATIONS.add("lombok.Value");
+        LOMBOK_ANNOTATIONS.add("lombok.RequiredArgsConstructor");
+        LOMBOK_ANNOTATIONS.add("lombok.AllArgsConstructor");
+        LOMBOK_ANNOTATIONS.add("lombok.NoArgsConstructor");
+        LOMBOK_ANNOTATIONS.add("lombok.Builder");
+    }
+
+    @Override
+    protected Collection<String> defaultSuppressionAnnotations() {
+        Collection defaultValues = new ArrayList<String>();
```

Comment:
ArrayList (and HashSet for example) have a constructor taking a collection. You could just write `return new ArrayList<>(LOMBOK_ANNOTATIONS);`